bauh/gems/arch/controller.py

```diff
@@ -3323,7 +3323,7 @@ def clean_cache(self, root_password: Optional[str], watcher: ProcessWatcher) ->
 
         if not cache_dir or not os.path.isdir(cache_dir):
             watcher.show_message(title=self.i18n['arch.custom_action.clean_cache'].capitalize(),
-                                 body=self.i18n['arch.custom_action.clean_cache.no_dir'.format(bold(cache_dir))].capitalize(),
+                                 body=self.i18n['arch.custom_action.clean_cache.no_dir {}'.format(bold(cache_dir))].capitalize(),
```

Comment:
This is actually my mistake, but if you are willing to provide the right fix: `self.i18n['arch.custom_action.clean_cache.no_dir'].format(bold(cache_dir))].capitalize()

Reply:
fixed in new commit